types/elastic-apm-node/tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "dtslint/dt.json",
```

Comment:
I'm guessing the build is unhappy about this trailing comma.

Reply:
Got rid of it and squashed! Thanks for the approval.